Cargo.toml

```diff
@@ -12,16 +12,7 @@ exclude = [
 ]
 
 [patch.crates-io]
-tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "romac/prost-dev" }
-tendermint-rpc          = { git = "https://github.com/informalsystems/tendermint-rs", branch = "romac/prost-dev" }
-tendermint-proto        = { git = "https://github.com/informalsystems/tendermint-rs", branch = "romac/prost-dev" }
-tendermint-light-client = { git = "https://github.com/informalsystems/tendermint-rs", branch = "romac/prost-dev" }
-
-bytes = { git = "https://github.com/tokio-rs/bytes", tag = "v0.6.0" }
-
-tonic = { git = "https://github.com/romac/tonic", branch = "romac/hyper-0.14-dev" }
-
-prost        = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-types  = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-build  = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-derive = { git = "https://github.com/danburkert/prost", branch = "master" }
+tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
```

Comment:
Maybe we don't need master, and can stick to RC3 ? https://github.com/informalsystems/tendermint-rs/commit/91bc1f6170528c82fb1aacc370e9d75d7e571bd2

Reply:
yeh, that would be nice

Reply:
Done